compiler/il/symbol/OMRResolvedMethodSymbol.cpp

```diff
@@ -69,6 +69,7 @@
 #include "ras/Debug.hpp"                        // for TR_Debug
 #include "runtime/Runtime.hpp"
 #include "infra/SimpleRegex.hpp"
+#include "optimizer/TransformUtil.hpp"
```

Comment:
Could we place this optimizer include in alphabetical order with the other optimizer includes?